src/store/reducers/movies.ts

```diff
@@ -76,16 +76,18 @@ const movies = createSlice({
 export const { addMoviesState, addMovies, pushMovie, putMovie, sortMovies  } = movies.actions
 export const { addMovie, setShowHeader, setOpenForm, setEditStatus } = movie.actions
 
+export const selectNotEmptyListMovies = (state: RootState) => state.movies.data.length > 0
+
 export const selectMovie = (state: RootState) => state.movie.data
 export const selectShowHeader = (state: RootState) => state.movie.isShowHeader
 export const selectOpenForm = (state: RootState) => state.movie.isOpenForm
 export const selectEditStatus = (state: RootState) => state.movie.isEdit
 
 export const selectListGenres = (state: RootState) => {
-    const listGenres = state.movies.data.map((movie: any) => {
+    const listGenres = state.movies.data.slice(0,2).map((movie: any) => {
         return movie.genres
     })
-    return [...new Set([].concat(...listGenres))]
+    return [...new Set(['Action', 'Fantasy', 'Romance'].concat(...listGenres))]
```

Comment:
shouldn't it be an empty array?

Reply:
here it is assumed that there will be 3 static genres, and the rest will be loaded depending on the current movies